### DISCUSSION TITLE
How do style selectors work?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [AvaloniaUI/Avalonia#16378] [Q&A]
How do style selectors work?
It seems like half the time things don't work.
Maybe that's because there are nested controls? 
But there needs to be a better api and documentation for this. 

```
    <Style Selector="TabItem">
        <Setter Property="Padding" Value="3"/>
        <Setter Property="FontSize" Value="12"/>
        <Setter Property="Background" Value="#3b3b3b"/>
        <Setter Property="BorderBrush" Value="#505050"/>
        <Setter Property="BorderThickness" Value="0,1,0,0"/>
    </Style>
    
```

Btw Background doesn't do anything for TabItem and using selectors like :is:pointerover does nothing.

**Comment by rabbitism** [comment]
Please read documentation first.



https://docs.avaloniaui.net/docs/guides/styles-and-resources/troubleshooting

**Reply by Shadowblitz16** [reply]
Ok cool that got most of it working... 
How do I color the tab control background strip though?
![image](https://github.com/user-attachments/assets/e6e2425d-2d39-4e28-aac4-af5d0e1c3ef9)
